Makefile

```diff
@@ -53,12 +53,14 @@ DECI2_DEBUG = 0
 CHILDPROOF = 0
 
 # ======== DO NOT MODIFY VALUES AFTER THIS POINT! UNLESS YOU KNOW WHAT YOU ARE DOING ========
+REVISION = $(shell $(($(cat DETAILED_CHANGELOG | grep "rev" | head -1 | cut -d " " -f 1 | cut -c 4- 2>/dev/null) + 1)))
+
 GIT_HASH = $(shell git rev-parse --short=7 HEAD 2>/dev/null)
 ifneq ($(shell git diff --quiet; echo $$?),0)
   DIRTY = -dirty
 endif
 
-OPL_VERSION = $(VERSION).$(SUBVERSION).$(PATCHLEVEL)$(if $(EXTRAVERSION),-$(EXTRAVERSION))$(if $(GIT_HASH),-$(GIT_HASH))$(if $(DIRTY),-$(DIRTY))$(if $(LOCALVERSION),-$(LOCALVERSION))
+OPL_VERSION = $(VERSION).$(SUBVERSION).$(PATCHLEVEL).$(REVISION)$(if $(EXTRAVERSION),-$(EXTRAVERSION))$(if $(GIT_HASH),-$(GIT_HASH))$(if $(DIRTY),-$(DIRTY))$(if $(LOCALVERSION),-$(LOCALVERSION))
```

Comment:
Yeah, that's what I suggested in the edit of my last answer on psx-scene - because of BatRastard post (rev count lost -/+1) and your comparaison with a fixed bitbucket changelog. Sorry if you had to revert changes (but hey, no hurry on this, it's not a life or death matter anyway).